examples/machine_learning/neural_network.cpp

```diff
@@ -18,6 +18,16 @@
 using namespace af;
 using std::vector;
 
+#define ASSERTLOG(TEST, MESSAGE) if (!(TEST)) std::cerr << MESSAGE << std::endl;
```

Comment:
Could you remove the ASSERTLOG from this example. I don't think we should have these messages in the examples.

Reply:
done